src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+ul {
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  font-size: 11.6px;
```

```suggestion
  font-size: 12px;
```
Чому така цифра? По дизайну шрифт має бути 12